tests/toranj/openthread-core-toranj-config.h

```diff
@@ -155,6 +155,15 @@
  */
 #define OPENTHREAD_CONFIG_ECDSA_ENABLE 1
 
+/**
+ * @def OPENTHREAD_CONFIG_DETERMINISTIC_ECDSA_ENABLE
+ *
+ * Define to 1 to generate ECDSA signatures determinsitically
+ * according to RFC 6979 instead of randomly.
+ *
+ */
```

Comment:
nit: Not needed, though it is fine to keep.

Reply:
@stevew817 , would you mind removing this note?

Reply:
@jwhui done.